### DISCUSSION TITLE
How do I suggest new Extension ideas or show interest in an Extension idea?

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to collect suggestions for new Docker Extensions. We hope that you:
  * Share ideas.
  * Engage with other community members on their ideas and explain how it could benefit you or your team.
  * Upvote 👍  ideas you'd love to see.
  * Get inspired and consider building a Docker Extension yourself. 

  To get started suggesting ideas:
  * Browse the existing discussions to make sure there isn't already a request.
  * Create a new discussion
  * Set the title as the name of the extension you want to see
  * In the body, please share why this is useful and what problems or challenges it will help you with.  The more information you can provide the better. We want to make sure that when an extension developer considers creating your idea, they have enough information to get started. 
  
  To show your enthusiasm for an idea, simply click the ⬆️ up arrow next to the idea.  If you have more information that could help someone interested in building the extension, add a comment.

**Comment by vjkrishn** [comment]
Any Virtual Machine

**Comment by cbarney813** [comment]
Need a Composer for this.

**Reply by cotyust123** [reply]
Trino DB software

**Reply by amyb12345** [reply]
@cotyust123 The best way to suggest a new extension idea is to start a new discussion item for it. I've created one for your request - https://github.com/docker/extension-ideas/discussions/166
Would you be able to add more info on how this will help you and what functionality is needed?  Thanks!

**Comment by cotyust123** [comment]
Trino DB software

**Comment by govindas** [comment]
I would like to take notes associated to a container

**Comment by Sqrooge** [comment]
Tautulli,

I was very surprised to see grafana but missing tautulli in your extensions...
so my suggestion would be to add that

**Comment by tonyw54** [comment]
I'm not sure if this is still monitored, but I would like a gitlab extension. Something that lets me connect to and manage my gitlab container images in my pipelines, directly from docker desktop.

**Comment by braeculp** [comment]
I would like a get some kind of microsoft extension

**Comment by iamprovidence** [comment]
An extension that automatically launches a specified Docker Compose stack or individual container whenever Docker Desktop starts
